internal/server/operations/create_test.go

```diff
@@ -51,6 +51,53 @@ func TestCreateServerParseSimple(t *testing.T) {
 	assert.Equal(age, int64(32))
 }
 
+func TestCreateServerParseInclude(t *testing.T) {
+	appDB := db.New()
+	db.AddSampleModels(appDB)
+	assert := assert.New(t)
+	req, err := http.NewRequest("POST", "/user.create", strings.NewReader(`{
+			"data":{
+				"firstName":"Andrew",
+				"lastName":"Wansley",
+				"age": 32
+		 	 },
+		 	 "include":{
+				"profile":true
+		  	 }
+		  }`))
+	if err != nil {
+		t.Fatal(err)
+	}
+	req = mux.SetURLVars(req, map[string]string{"object": "user"})
+
+	qs := CreateServer{}
+	ctx := middleware.NewRWTxContext(context.Background(), appDB.NewRWTx())
+	parsed, err := qs.Parse(ctx, req)
+	if err != nil {
+		t.Error(err)
+	}
+	parsedReq := parsed.(CreateRequest)
+
+	assert.IsType(parsedReq, CreateRequest{})
+
+	op := parsedReq.Operation
+	rec := op.Record
+    inc := parsedReq.Include.Includes
```

Comment:
run gofmt (or integrate that into your text editor) to clean up this code (indentation etc)